nvflare/private/fed/server/server_engine.py

```diff
@@ -396,8 +400,8 @@ def shutdown_server(self) -> str:
 
         touch_file = os.path.join(self.args.workspace, "shutdown.fl")
         _ = self.executor.submit(lambda p: server_shutdown(*p), [self.server, touch_file])
```

Comment:
This place is different from other place directly using a single Thread. Because the "self.server" could be a list of multiple servers. This is consistent with the implementation all the other places.